Exercise_B/index.css

```diff
@@ -2,3 +2,27 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+
+.blueBox {
+    height: 100px;
+    width: 100px;
+    background-color: blue;
+    float left;
+    display: inline-block;
+}
+
+.redBox {
+    height: 100px;
+    width: 100px;
+    background-color: red;
+    margin-left: 45%;
+}
+
+.greenBox {
+    height: 100px;
+    width: 100px;
+    background-color: green;
+    float: right;
+    display: inline-block;
+}
```

Comment:
Decent implementation for this exercise, though you can achieve this without using margins.
On your red class only have the background colour and:
clear: both;
margin: auto;

But I would extract the width and height into a separate class such that you do not repeat the properties over and over again for each block.

Consider using a box class and id's for blue, red and green